modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -97,4 +97,124 @@ Status description:: error: connection exception - alias chains are not permitte
+
+[[_data-exceptions]]
+== Data exceptions
+
+The following is a list of the data exceptions that can occur in Neo4j.
+
+=== 22N00 
+
+Status description:: error: data exception - unsupported value. The provided value is unsupported and cannot be processed.
+
+=== 22N01 
+
+Status description:: error: data exception - invalid type.
+Expected the value { $value } to be of type { $valueTypeOredList }, but was of type { $valueType }.
+
+=== 22N02 
+
+Status description:: error: data exception - specified negative numeric value. Expected { $option } to be a positive number but found { $value } instead.
+
+=== 22N03 
+
+Status description:: error: data exception - specified numeric value out of range. Expected { $component } to be of type { $valueType } and in the range { $lower }  to { $upper } but found { $value }.
+
+=== 22N04 
+
+Status description:: error: data exception - invalid input value. Invalid input { $input } for { $context }. Expected one of { $inputOredList }.
+
+=== 22N05
+
+Status description:: error: data exception - input failed validation. Invalid input { $input } for { $context }.
+
+=== 22N06 
+
+Status description:: error: data exception - empty input string. Invalid input. { $option } needs to be specified.
+
+=== 22N07 
+
+Status description:: error: data exception - invalid pre-parser option key. Invalid pre-parser option(s): { $optionList }.
+
+=== 22N08 
+
+Status description:: error: data exception - invalid pre-parser combination. Invalid pre-parser option, cannot combine { $option1 } with { $option2 }.
+
+=== 22N09 
+
+Status description:: error: data exception - conflicting pre-parser combination. Invalid pre-parser option, cannot specify multiple conflicting values for { $option }.
+
+=== 22N10 
+
+Status description:: error: data exception - invalid pre-parser option value. Invalid pre-parser option, specified { $input } is not valid for option { $option }. Valid options are: { $optionAndedList }.
+
+=== 22N11 
+
+Status description:: error: data exception - invalid argument. Invalid argument: cannot process { $input }.
+
+=== 22N12 
+
+Status description:: error: data exception - invalid date, time, or datetime format. Invalid argument: cannot process { $input }.
+
+=== 22N13 
+
+Status description:: error: data exception - invalid time zone. Specified time zones must include a date component.
+
+=== 22N14 
+
+Status description:: error: data exception - invalid temporal value combination. Cannot select both { $temporal } and { $component }.
+
+=== 22N15 
+
+Status description:: error: data exception - invalid temporal component. Cannot read the specified { $component } component from { $temporal }.
+
+=== 22N16 
+
+Status description:: error: data exception - invalid import value. Importing entity values to a graph with a USE clause is not supported. Attempted to import { $expr } to { $graph }.
+
+=== 22N17 
```

Comment:
We have deleted 22N17 from the code and it also is marked as 'DELETE' in the spreadsheet. I think it was a copy of another code or wrong in some other way. Maybe you can remind me @anderson4j ? Anyway, we should probably exclude this one from docs too.